yolo.go

```diff
@@ -19,7 +19,7 @@ func main() {
     command := args[0]
 
     if startsWithAnyOf(command, []string { "mvn", "./mvnw", "mvnd" }) {
-        args = append(args, "-DskipTests")
+        args = append(args, "-DskipTests", "-Dspotbugs.skip=true", "-Dcheckstyle.skip")
```

Comment:
Question: `-Dcheckstyle.skip` or should it be `-Dcheckstyle.skip=true` ?

Reply:
Both options work for me